dandischema/metadata.py

```diff
@@ -90,3 +95,50 @@ def validate_asset_json(data, schema_dir):
     with Path(schema_dir, "asset.json").open() as fp:
         schema = json.load(fp)
     jsonschema.validate(data, schema)
+
+
+def validate(obj, schema_version=None, schema_key=None):
```

Comment:
This docstring isn't terminated.

Reply:
yup - will be fixed in the update. got sidetracked on improving models.